shop/src/main/java/com/example/shop/controller/OrderController.java

```diff
@@ -0,0 +1,78 @@
+package com.example.shop.controller;
+
+
+import com.example.shop.Entity.Product;
+import com.example.shop.Repository.InProductRepo;
+import com.example.shop.dto.AddressDTO;
+import com.example.shop.dto.OrderDTO;
+import com.example.shop.dto.OrderDetailDTO;
+import com.example.shop.dto.OrderrObjDTO;
+import com.example.shop.service.OrderServiceImpl;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+
+public class OrderController {
+    private final OrderServiceImpl srv;
+    private final InProductRepo productRepo;
+
+    public OrderController(OrderServiceImpl srv, InProductRepo productRepo) {
```

Comment:
You don't need the constructor here. Spring will take care of the service and repo instances (Dependency Injection).